web/app.js

```diff
@@ -451,6 +453,16 @@ const PDFViewerApplication = {
         : null;
     }
 
+    let editorUndoBar = null;
+    if (appConfig.editorUndoBar) {
+      editorUndoBar = new EditorUndoBar(
+        appConfig.editorUndoBar,
+        eventBus,
+        l10n
+      );
+    }
+    this.editorUndoBar = editorUndoBar;
```

Comment:
This can now be simplified, see also the next comment.
```suggestion
    if (appConfig.editorUndoBar) {
      this.editorUndoBar = new EditorUndoBar(
        appConfig.editorUndoBar,
        eventBus
      );
    }
```